.gitlab-ci.yml

```diff
@@ -445,93 +445,98 @@ validate:edge+flambda:
     OPAM_SWITCH: edge
     OPAM_VARIANT: "+flambda"
 
-ci-aac_tactics:
-  <<: *ci-template
+# Libraries are by convention the projects that depend on Coq
+# but not on its ML API
 
-ci-bedrock2:
+library:ci-bedrock2:
   <<: *ci-template
   allow_failure: true
```

Comment:
I guess this started working again at some point, if so then we should remove the allow_failure.

Reply:
@ejgallego I'm removing that second commit then.